src/client.rs

```diff
@@ -64,6 +66,16 @@ fn handle_list_disks(s: &mut Socket) {
     };
 }
 
+fn handle_jira_tickets(s: &mut Socket) -> BynarResult<()>{
+    trace!("start of test get_jir_tickets");
+    let tickets = helpers::get_jira_tickets(s)?;
+    trace!("End of test get_jir_tickets");
```

Comment:
Same with this.  Something like: `handle_jira_tickets finished` or some similar language.

Reply:
Done